rktl_launch/launch/car.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
+    <arg name="car_name" default="car0" />
+    <arg name="real" default="true" />
+
+    <group ns="$(arg car_name)/">
+        <!-- Filter -->
+        <include file="$(find rktl_control)/launch/mean_odom_filter.launch" />
+
+        <group if="$(arg real)">
+            <!-- Control -->
+            <include file="$(find rktl_control)/launch/velocity_controller.launch" />
+
+            <!-- Hardware -->
+            <include file="$(find rktl_control)/launch/hardware_interface.launch" />
+        </group>
+    </group>
+</launch>
```

Comment:
Like this. Given that all these files are in `rktl_control`, I think this launch file should exist there. That way control can be modified independently.